pkg/collector/corechecks/networkpath/networkpath.go

```diff
@@ -0,0 +1,134 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package networkpath defines the agent corecheck for
+// the Network Path integration
+package networkpath
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/DataDog/datadog-agent/comp/forwarder/eventplatform"
+	"github.com/DataDog/datadog-agent/pkg/aggregator/sender"
+	"github.com/DataDog/datadog-agent/pkg/autodiscovery/integration"
+	"github.com/DataDog/datadog-agent/pkg/collector/check"
+	core "github.com/DataDog/datadog-agent/pkg/collector/corechecks"
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/networkpath/traceroute"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"github.com/DataDog/datadog-agent/pkg/util/optional"
+)
+
+// CheckName defines the name of the
+// Network Path check
+const CheckName = "networkpath"
```

Comment:
Most integrations in integrations-core have snake case names when it's a multiple words name: https://github.com/DataDog/integrations-core

Shall we use `network_path` instead of `networkpath`?

```suggestion
const CheckName = "network_path"
```

Reply:
1/ I agree, makes sense to update the name to `network_path`

2/ I don't think we should rename the folders. It would be inconsistent with what I'm seeing in all the other corechecks in the agent repo i.e. `container_image` check's package name is `containerimage`. IIRC all one word and lowercase is the idiomatic way to handle multi-word package names in Go as well

For metrics I'm on board for `network_path.`

Reply:
Agree with ☝️ 

Thanks :)